Integrations/integration-ATD.yml

```diff
@@ -499,7 +510,7 @@ script:
       - txt
       - xml
       - zip
-      - json
+      - table
```

Comment:
need to keep json for backward compatibility (for users that already used it in their playbooks)